client/jetpack-cloud/sections/pricing/jpcom-masterbar/style.scss

```diff
@@ -735,6 +740,24 @@
 		animation-name: submenu-item;
 		animation-duration: 0.25s;
 		animation-timing-function: ease-in-out;
+
+		@for $column from 1 through 5 {
+			&:nth-of-type(#{$column}) {
+				animation-delay: calc(
+					calc(#{$delay} * #{$column}) + calc(#{$duration} / #{$delay-ratio})
+				);
+
+				@for $i from 1 through 6 {
+					.header__submenu-links-list > li:nth-of-type(#{$i}) > a,
+					.header__submenu-links-list > li:nth-of-type(#{$i}) > p {
+						animation-delay: calc(
+							calc(#{$delay} * #{$i + 1 + $column}) +
+								calc(#{$duration} / #{$delay-ratio})
+						);
+					}
+				}
+			}
```

Comment:
Nice refactor! Using Scss @for loops!! 🙌

Reply:
Can't take too much credit, I mostly copied from jetpack.com and I made the original, terrible, SCSS here 😆 But thank you!